ui/src/templates/components/CommunityTemplateImportOverlay.tsx

```diff
@@ -89,7 +89,9 @@ const mstp = (state: AppState, props: Props): StateProps => {
     org,
     templateName: props.params.templateName,
     flags: state.flags.original,
-    resourceCount: getTotalResourceCount(state),
+    resourceCount: getTotalResourceCount(
+      state.resources.templates.communityTemplateToInstall.summary
```

Comment:
This seems like a really deep get, are there any guards and/or a reasonable default here?

Reply:
it definitely is, good call. `state.resources.templates` is taken care of at the resources level, `communityTemplateToInstall` is taken care of having a default shape here: https://github.com/influxdata/influxdb/pull/18852/files#diff-03c3cd4b5faeff2eaea0b8c3c19df44bR28